### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by petr-balogh**

### CODE REVIEW

**Comment by petr-balogh** [comment]
Just attempt to fix doc with adding new support for .md files mentioned in doc.

**Review by DanielOsypenko — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

File: ocs_ci/ocs/cluster.py [hunk 1]
```diff
@@ -67,8 +67,10 @@ class CephCluster(object):
     This class has depiction of ceph cluster. Contains references to
     pod objects which represents ceph cluster entities.
 
+    Parameters
+    ----------
```
Comment (dahorak) [inline ocs_ci/ocs/cluster.py:71]: Is this intentionally here?

**Review by dahorak — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Review by mashetty330 — APPROVED** [review]

**Review by sagihirshfeld — APPROVED** [review]

**Pull request merged**